.github/workflows/pull_request.yml

```diff
@@ -27,9 +27,9 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install -e .
-          pip install -r test_requirements.txt
+          pip install numpy wheel \
```

Comment:
why does ut need extra numpy and wheel? `pip install -e .` will install numpy, and wheel is for packaging. Please recover this file or let me know the problem of missing these too.

BTW `|` in yml will include new lines, so the `&&` and `\` has no need too.

Reply:
It seems that the problem still exists when using python3.8. The only thing I know is that bfloat16==1.1 depends on numpy.

Reply:
I see there's a bug in bfloat16==1.1, and they release 1.2.0 to solve this, however 1.2.0 only support python>=3.9. python3.8 can only install the `bfloat16.tar.gz` to compile it locally

It's a bad managed package and I suggest to remove bfloat16 in ut.